pkgs/applications/networking/browsers/kristall/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, qmake, qtbase, qtmultimedia, wrapQtAppsHook }:
+
+stdenv.mkDerivation rec {
```

Comment:
I believe it is preferable to use `mkDerivation` in place of `stdenv.mkDerivation` (and add `mkDerivation` to the argument list).

See https://hydra.nixos.org/build/96804884/download/1/nixpkgs/manual.html#sec-language-qt

Reply:
Thanks, that is is better.